File: tests/generators/ios/objc/files/MethodRequest.m
  PostRequest *obj = [NSEntityDescription insertNewObjectForEntityForName:@"PostRequest" inManagedObjectContext:sharedMgr.managedObjectStore.mainQueueManagedObjectContext];
  obj.body = body;
  obj.title = title;
  [sharedMgr postObject:obj path:@"post/" parameters:nil success:success failure:failure];

File: tests/generators/ios/swift/files/MethodRequest.swift
    let obj = NSEntityDescription.insertNewObjectForEntityForName("PostRequest", inManagedObjectContext: sharedMgr.managedObjectStore.mainQueueManagedObjectContext) as! PostRequest
    obj.body = body
    obj.title = title
    sharedMgr.postObject(obj, path: "post/", parameters: nil, success: success, failure: failure)
